configs/components/oifs/oifs.yaml

```diff
@@ -201,6 +202,7 @@ compile_infos:
         comp_command: cd make; ../fcm/bin/fcm make -v -j8 -f oifs.cfg
         git-repository: https://gitlab.dkrz.de/modular_esm/oifs-40r1.git
         install_bins: make/esm/oifs/bin/master.exe
+        source_code_permissions: 700
```

Comment:
No, it cannot, because the use of the `esm_parser` of `esm_master` is very limited. Also, the fact that we have `compile_infos` is not ideal. Anyway, this line is unnecessary, I tested and with the one above only, everything seems to work, so in order to avoid duplication, I have deleted this one.No, it cannot, because the use of the `esm_parser` of `esm_master` is very limited. Also, the fact that we have `compile_infos` is not ideal. Anyway, this line is unnecessary, I tested and with the one above only, everything seems to work, so in order to avoid duplication, I have deleted this one.